### PR TITLE
(DOCSP-19629): Configure Prettier and ESLint for Web SDK tutorial

### PR DESCRIPTION
**Pull request by mongodben**
## Pull Request Info

Added Prettier and ESLint configuration to the Web SDK tutorial to normalize code formatting across the project and enforce standard practices. 

Note that this expands the scope of the initial ticket request, which was just to run prettier on the repo. Based on the changes in this PR, coding practices should be enforced, so it's no longer necessary to manually run prettier. 

### Jira

- https://jira.mongodb.org/browse/DOCSP-19629

### Staged Changes (Requires MongoDB Corp SSO)

- N/A

### Review Guidelines

[REVIEWING.md](https://github.com/mongodb/docs-realm/blob/master/REVIEWING.md)

### CODE REVIEW

**Review by cbush — CHANGES_REQUESTED** [review]
Why did you make a specific prettierrc when we can just use the standard? The whole point is avoiding arguments about personal preferences.

File: tutorial/web/.prettierrc.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "trailingComma": "none",
```
[the remark this answers is not in the document]
Comment (mongodben) [reply tutorial/web/.prettierrc.json:2]: personal preference 🤷. i can change to make it with commas

[the remark this answers is not in the document]
Comment (mongodben) [reply tutorial/web/.prettierrc.json]: changes to make `"trailingComma": "all",`

[the remark this answers is not in the document]
Comment (cbush) [reply tutorial/web/.prettierrc.json]: Why not just remove the prettierrc though?

[the remark this answers is not in the document]
Comment (mongodben) [reply tutorial/web/.prettierrc.json]: yep looks like we can do that. i believe i put like this b.c i was following along w some tutorial. 

upon reading through the [prettier default options docs](https://prettier.io/docs/en/options.html), the only difference btwn what's here currently and the default is the default is `"trailingComma": "es5"`. there'd be no practical difference btwn them here b.c the [only difference for these settings relates to typescript](https://prettier.io/docs/en/options.html#trailing-commas).

will remove this file and validate that everything still works as expected.

[the remark this answers is not in the document]
Comment (mongodben) [reply tutorial/web/.prettierrc.json]: removed the `.prettierrc` file and associated changes.

**Review by cbush — APPROVED** [review]

File: tutorial/web/.eslintrc.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{
+  "parserOptions": {
+    "ecmaVersion": 11,
+    "sourceType": "module"
+  },
+  "rules": {
+    "comma-dangle": ["always-multiline"]
```
Comment (cbush) [inline tutorial/web/.eslintrc.json:7]: Is this needed?

Comment (mongodben) [reply tutorial/web/.eslintrc.json:7]: yep was needed b/c prettier was conflicting w the eslint config. this fixed.

Comment (cbush) [reply tutorial/web/.eslintrc.json:7]: This shouldn't be needed. Could you retry without it now that the prettierrc is gone?

Comment (mongodben) [reply tutorial/web/.eslintrc.json:7]: sure thing. i think i already did. but maybe this is exposing a deeper issue with the setup that was patched overby adding these rules

**Comment by mongodben** [comment]
since this PR and associated ticket are already closed, creating new ticket to track - https://jira.mongodb.org/browse/DOCSP-19864

**Pull request merged**